[#1] First cut on an alpha prototype

kardan opened the pull request:
- Added a simple REST API with Liberator
- Added basic message queue event handler
- Created a crude namespace data-store for fixture data
- Made sure the app can run from lein and as a jar
- Updated README with more content

kardan committed:
[#1] First cut on a prototype

- Added a simple REST API with Liberator
- Added basic message queue event handler
- Created a crude namespace data-store for fixture data
- Made sure the app can run from lein and as a jar
- Updated README with more content

iperdomo commented:
As discussed in our conversation:
- Amend the copyright year
- Routes shouldn't contain a trailing slash
- Double check the usage of spaces vs tabs on `project.clj`

kardan committed:
[#1] Fixes after code review

- amended copyright year in the file header
- removed trailing slashes on routes
- fixed an indentation issue in projects.clj

kardan commented:
Updated "feature/1-setup" after review.

kardan commented:
Honestly I don't think it is "pernickety", it's the services' interface (ok, maybe a bit - but it's still important ;-) ). I initially added slashes while I was investigating how strict Liberator was with them and forgot to remove them. So I'm in the non trailing slash religion, but got caught slashing...

I don't think it's a matter of being pedant with URLs, but more about preference. One could just as well argue that /services returns "a" resource that list the services. And that the slashes represent the hierarchy up to that resource. It's more common with APIs, but even blogs sometimes use of non trailing slashes on a [category list](http://roy.gbiv.com/untangled/tag/opensolaris).

iperdomo commented on README.md:
```diff
@@ -1,8 +1,45 @@
+## REST API
+Available routes, you will need trailing slashes.
+
+/services/
```
the documentation still has the trailing slash

iperdomo commented on src/akvo/notifications/rest_api.clj:
```diff
@@ -0,0 +1,52 @@
+            [ring.middleware.params :refer [wrap-params]]
+            [akvo.notifications.data-store :as data]))
+
+(def available-media-types
```
are we going to support all this media types?

kardan replied:
No, I only added a couple as an example. I guess future discussions about serialisation and architecture will steer what types we expose.

iperdomo commented:
:+1: code reviewed + correct
